packages/host/app/resources/card-type.ts

```diff
@@ -31,21 +35,32 @@ export interface Type {
   displayName: string;
   super: Type | undefined;
   fields: { name: string; card: Type | CodeRef; type: FieldType }[];
+  moduleMeta: {
+    extension: string;
+    realmInfo: RealmInfo;
+  };
   codeRef: CodeRef;
 }
 
 export class CardType extends Resource<Args> {
   @tracked type: Type | undefined;
+  @service declare cardService: CardService;
   declare loader: Loader;
   typeCache: Map<string, Type> = new Map();
+  moduleMetaCache: Map<string, { extension: string; realmInfo: RealmInfo }> =
```

Comment:
perhaps the cache would actually do more if it lived in module scope--then all resources could share it. I would think this kind of information would change very rarely if ever. so probably a cache that could be shared be all resource instances would be more beneficial

Reply:
https://linear.app/cardstack/issue/CS-6056/remove-modulemetacache-from-cardtype-place-cache-at-module-scope

Actually I think the realmInfoService can be used for this